sdk/modelsrepository/Azure.IoT.ModelsRepository/src/Azure.IoT.ModelsRepository.csproj

```diff
@@ -18,6 +18,7 @@
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
     <PackageReference Include="System.Text.Json" />
+    <PackageReference Include="System.Text.Encodings.Web" />
```

Comment:
would you please add a comment here to indicate why we are taking a direct dependency to this?

Reply:
Done.